src/main/java/de/tum/in/www1/artemis/service/connectors/bitbucket/BitbucketService.java

```diff
@@ -256,7 +236,92 @@ public void createUser(String username, String password, String emailAddress, St
     }
 
     /**
-     * Adds an Bitbucket user to (multiple) Bitbucket groups
+     * Updates a user on Bitbucket
+     *
+     * @param username     The username of the user
+     * @param emailAddress The new email address
+     * @param displayName  The new display name
+     * @throws BitbucketException
+     */
+    public void updateUserDetails(String username, String emailAddress, String displayName) throws BitbucketException {
+        UriComponentsBuilder userDetailsBuilder = UriComponentsBuilder.fromHttpUrl(bitbucketServerUrl + "/rest/api/latest/admin/users");
+        Map<String, Object> userDetailsBody = new HashMap<>();
+        userDetailsBody.put("name", username);
```

Comment:
We often use DTOs here, is there a special reason why none is used here?

Reply:
Because it's one special use for one external endpoint. I think we wouldn't need the DTO for anything else so it would only have one usuage. That's why I deemed a DTO to be overkill in this situation. Do think otherwise?

Reply:
For me that's fine, I was just not sure if we have a convention on when to use a DTO.